packages/extension-ui/src/Popup/Signing/Request/index.tsx

```diff
@@ -126,14 +128,14 @@ export default function Request ({ account: { accountIndex, addressOffset, isExt
           error={error}
           isExternal={isExternal}
           isFirst={isFirst}
-          isSignable
           setError={setError}
           signId={signId}
         />
       </>
     );
   } else if (hexBytes !== null) {
     const { address, data } = request.payload as SignerPayloadRaw;
+    const account = accounts.find((account) => decodeAddress(account.address).toString() === decodeAddress(address).toString());
```

Comment:
This is quite a heady operation, since it uses decodeAddress (twice). This should be memo-ized.